finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+
+If the balance is zero, it means the following:
+
+* For the preliminary account, the money is moved from the merchant's preliminary account to their liquid account.
+* For the liquid account, the money is paid out from Zettle to the mechant's bank account.
+<!-- Returns accumulated balance at a specific point in time. After a deposit has been made from LIQUID account to merchant’s bank account (daily, monthly or weekly), the balance is usually zero. If not, it will return the amount that is still due in merchant’s Zettle account. -->
+
+```
+GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/balance
+```
+
+See example [Fetch balance for a liquid account](#fetch-balance-for-a-liquid-account).
+
+### Parameters
+
+<details open="true">
+<summary>Parameters</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|accountTypeGroup |string |path |required |The account type from which the data like transactions is retrieved. You can use one of the following account types: <br/><ul><li> `PRELIMINARY` account where transactions are to be confirmed by third-party acquirers.</li><li> `LIQUID` account where transactions are to be paid out to the merchant.</li></ul>
+|at |string |query |optional |Used to fetch account balance that is available at a time in the UTC time zone. If it's used, any transaction after that point will be ignored. If it's not used, the balance of all transactions at the current point of time is returned. <br/>You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the account balance is successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the account balance.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request .  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account balance at a point in time. It's represented by `totalBalance` and `currencyId`.
+|totalBalance |integer |The account balance. It can be negative. For example, `-300`
```

Comment:
Should we mention that it is not fractionated?
If we return 
```
{ 
totalBalance: -300,
currencyId: "SEK"
}
```
that means -3,00 SEK.

Reply:
YES 😃  definitely! Very good point.